src/classes/BDI_DataImportDeleteBTN_CTRL.cls-meta.xml

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<ApexClass xmlns="http://soap.sforce.com/2006/04/metadata">
+    <apiVersion>39.0</apiVersion>
```

Comment:
@davidhabib , the new controller and VF page API versions are set to the latest API version (as indicated in the "PR Checklist" document). The package.xml version is 37.0 however. Can you please check into what version we should use for the new files?